lib/organization.js

```diff
@@ -145,24 +145,22 @@ export class Organization {
   }
 
   /**
-   * returns
-   * @param name linke "SchemeRegistrar"
+   * returns schemes currently registered into this DAO, as Array<OrganizationSchemeInfo>
+   * @param name like "SchemeRegistrar"
    */
   async schemes(name) {
     // return the schemes registered on this controller satisfying the contract spec
-    // return all schems if contract is not given
+    // return all schemes if contract is not given
     const schemes = await this._getSchemes();
     if (name) {
-      return schemes.filter(s => {
-        return s.name === name;
-      });
+      return schemes.filter(s => s.name === name);
     } else {
       return schemes;
     }
   }
 
   /**
-   * returns schemes currently in this Organization as Array<OrganizationSchemeInfo>
+   * returns schemes currently in this DAO as Array<OrganizationSchemeInfo>
```

Comment:
why this change to DAO? seems confusing to use both, we should pick one or the other

Reply:
agreed, done